sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -1055,7 +1073,22 @@ alterColumnAction
     | dropDefault=DROP DEFAULT
     ;
 
+stringLit
```

Comment:
What is `Lit` in `stringLit`?

Reply:
there is an existing rule called stringLiteral (which is only used a couple of times).
So Lit is just a short form of Literal to avoid collision.
I am not picky about the name.....

Reply:
Ah I see. Honestly I was at the beginning reading this as `string list` :) 

If it's me I could name it `STRING_OR_ DOUBLEQUOTED_STRING` but which might be too long. Let's see if other person has an opinion on this naming.